### ISSUE TITLE
no track recording in 3.9.3 iOS

### ISSUE THREAD

**Issue by EugeneZmeuk** [osmandapp/OsmAnd-iOS#1110]
since my latest update to 3.9.3 on iOS track recording stopped working.
The current position is shown on the map correctly, but the tracks
always stay empty.

I already tried:
* put all plugin-settings to minimum values, except for accuracy
* reset all plugin-settings to default values
* close and restart the app
* deactivate and re-activate the plugin, with and without restart
of the app
* check system settings for app's access to location data

All without success.

**Comment by EugeneZmeuk** [comment]
There is the same issue for iPhone 6S (OsmAnd 3.90.4)

**Comment by EugeneZmeuk** [comment]
I tried to reinstall OsmAnd, but Rec feature doesn't work.
Next, I made a reset for settings and routing in iOS settings:
Settings -> OsmAnd Maps -> OsmAnd maps settings (Reset routing and Reset settings) (All my app profiles are deleted after that). 
Switch on Track recording: menu - Maps & Resources - Plugins - Track recording
Track recording feature is writing my trips now without an old issue.
